src/services/doppler-legacy-client.ts

```diff
@@ -306,13 +308,49 @@ export interface DopplerLegacyUpgradePlanContactModel {
 export class HttpDopplerLegacyClient implements DopplerLegacyClient {
   private readonly axios: AxiosInstance;
   private readonly baseUrl: string;
+  private window: any;
 
-  constructor({ axiosStatic, baseUrl }: { axiosStatic: AxiosStatic; baseUrl: string }) {
+  private isEnabledForRetry(error: AxiosError) {
+    if (error.config.url === 'WebAppPublic/Login') {
+      return true;
+    } else {
+      return false;
+    }
+  }
```

Comment:
```
  private isEnabledForRetry(error: AxiosError) {
    return error.config.url === 'WebAppPublic/Login';
  }
```

And, if you are not sure of having config:

```
  private isEnabledForRetry(error: AxiosError) {
    return error.config && error.config.url === 'WebAppPublic/Login';
  }
```

Reply:
Changed.